pages/02.reference/01.chat/05.ChatViewerList/doc.md

```diff
@@ -0,0 +1,59 @@
+# Chat Viewer List V2 Endpoints
+
+## GET /api/v2/chats/{channelId}/users
+
+### Request
+
+#### URI Parameters
+* channelId: uint - The ID of the channel you wish to chat in.
+
+#### Query Parameters
+
+* limit: uint? [1≤integer≤100] - Amount of items per page.
+* showLurkers: boolean? - Show lurkers in the channel for staff user (Authed API and requires JWT Token)
+* continuationToken: string? - ContinuationToken to paginate through all chat users in the channel. See [Continuation Tokens](TODO) for more info on how these work
```

Comment:
I would mention as this is an opaque token and suggest that you don't make assumption etc.

Reply:
Awesome

Reply:
I might not even document it and instead say "just use the link header" 😛